Makefile

```diff
@@ -389,6 +389,15 @@ bench_pbs: install_rs_check_toolchain
 	--bench pbs-bench \
 	--features=$(TARGET_ARCH_FEATURE),boolean,shortint,internal-keycache,$(AVX512_FEATURE) -p tfhe
 
+.PHONY: parse_integer_benches # Run python parser to output a csv containing integer benches data
+parse_integer_benches:
+	if [[ "$${BENCH_CSV_FILE}" == "" ]]; then \
```

Comment:
Does this double $ implies that you fetch this variable value from the environment ?

Reply:
yes, you need to escape the dollar to have the "true" dollar sign for the script, not sure if environment is better or not 🤔 wdyt ?

Reply:
It just seems odd since all other variable are handled at the beginning of the Makefile.
I would go for the classic way `BENCH_CSV_FILE="foo.csv" make parse_integer_benches`

Reply:
I set the variable at the start of the workflow like the other file required for slab to have the env var available for parsing and upload

Reply:
Yes but you can still do in the workflow something like `BENCH_CSV_FILE=${{ env.BENCH_CSV_FILE }} make parse_integer_benches`

Reply:
alright will take a second look